api/controllers/case.js

```diff
@@ -305,16 +307,42 @@ router.put("/:caseId", function editCaseById(req, res) {
  *
  */
 
-router.get("/:caseId", function getCaseById(req, res) {
+function getCaseById(req, res) {
   db
     .one(sql("../sql/case_by_id.sql"), {
       caseId: req.params.caseId,
       lang: req.params.language || "en"
     })
     .then(function(the_case) {
-      res.status(200).json({
-        OK: true,
-        data: the_case
+      getUserIfExists(req, res, function(userId) {
+        if (userId) {
+          db
+            .one(
+              "select * from bookmarks where bookmarktype = $1 AND thingid = $2 AND userid = $3",
+              ["case", req.params.caseId, userId]
+            )
+            .then(function(data) {
+              console.info("CASE IS BOOKMARKED");
+              the_case["bookmarked"] = true;
```

Comment:
This would be good to move into `[type]_by_id.sql`, but I can take care of that later.